test/cypress/dashboards/integration/EditMode/sidebar.cy.js

```diff
@@ -19,13 +16,42 @@ describe('Edit Mode sidebar', () => {
 
     it('Input should be empty if the title, caption or subtitle disabled, #21079.', () => {
         cy.toggleEditMode();
+        cy.openCellEditSidebar('#dashboard-col-1');
 
-        // Open sidebar
-        cy.get('#dashboard-col-1').click();
-        cy.get('.highcharts-dashboards-edit-toolbar-cell').children().eq(0).click();
         cy.get('.highcharts-dashboards-edit-accordion-header-btn').eq(1).click();
 
         // Assess
         cy.get('.highcharts-dashboards-edit-accordion-content input').should('have.value', '')
     });
+
+    it('Should be able to confirm or cancel changes, #20756.', () => {
```

Comment:
Can you please add test coverage for the visibility of the mask throughout the whole process?